ScrutinyPlot/plot3periods.py

```diff
@@ -0,0 +1,59 @@
+# Takes three file names, for full period, half of period, and fraction of period
+
+import ROOT
+import sys
+from DataFormats.FWLite import Events, Handle
+from ROOT import gStyle
+
+# Make VarParsing object
+# https://twiki.cern.ch/twiki/bin/view/CMS/SWGuideAboutPythonConfigFile#VarParsing_Example
+# from FWCore.ParameterSet.VarParsing import VarParsing
+# options = VarParsing ('python')
+# options.parseArguments()
+
+def FillHisto(filnum, histo):
+	infile = open(sys.argv[filnum], "r")
+	for inline in infile:
+		entries = inline.split(",")
+		# print entries
+		if len(entries[1]) > 0 and len(entries[4]) > 0 :
+			szintb = float(entries[1]) / 1024.0 / 1024.0 / 1024.0 / 1024.0
+			histo.Fill(float(entries[4]), szintb)
+	infile.close()
+
+
+scrutplot1 = ROOT.TH1F ("scrutiny1", "Dataset Usage for June 16 - December 9, 2017", 15, 0.0, 15.0)
+scrutplot2 = ROOT.TH1F ("scrutiny2", "", 15, 0.0, 15.0)
+scrutplot3 = ROOT.TH1F ("scrutiny3", "", 15, 0.0, 15.0)
+scrutplot1.SetStats(0)
+scrutplot2.SetStats(0)
+scrutplot3.SetStats(0)
+FillHisto(1, scrutplot1)
+FillHisto(2, scrutplot2)
+FillHisto(3, scrutplot3)
+scrutplot1.GetXaxis().SetTitle("Number of Accesses");
+scrutplot1.GetYaxis().SetTitle("Aggregated Data Size [TB]");
```

Comment:
My understanding that we need to use PB scale for T1+T2 sites, so it's better to use one common scale. Also it would be nice to produce both normal plot and log scale.